src/agent/onefuzz/Cargo.toml

```diff
@@ -44,14 +44,16 @@ onefuzz-telemetry = { path = "../onefuzz-telemetry"}
 stacktrace-parser = { path = "../stacktrace-parser" }
 backoff = { version = "0.3", features = ["tokio"] }
 
-[target.'cfg(target_os = "windows")'.dependencies]
+[target.'cfg(target_family = "windows")'.dependencies]
 winreg = "0.9"
 input-tester = { path = "../input-tester" }
 debugger = { path = "../debugger" }
 
-[target.'cfg(target_os = "linux")'.dependencies]
+[target.'cfg(target_family = "unix")'.dependencies]
 cpp_demangle = "0.3"
 nix = "0.21"
+
+[target.'cfg(target_os = "linux")'.dependencies]
 pete = "0.5"
-proc-maps = "0.1"
 rstack = "0.3"
+proc-maps = "0.1"
```

Comment:
Was this re-ordered on purpose, or just incidentally?

Reply:
incidental